src/clients/inst_autosetup.rb

```diff
@@ -454,6 +462,17 @@ def autosetup_users
       end
     end
 
+    # Import security settings from profile
+    def autosetup_security
+      security_config = Profile.current["security"]
+      if security_config
+        # Do not start it in second installation stage again.
+        # Writing will be called in inst_finish.
+        Profile.remove_sections("security")
+        Call.Function("security_auto", ["Import", security_config])
```

Comment:
I do not know how AY usually checks for errors, but shouldn't we check for the result here?

Reply:
Normally, not. There are a lot of modules where the return value will not be checked. It makes also no really sense here.